### ISSUE TITLE
Modify Robots Plugin to handle robots.txt

### ISSUE THREAD

**Issue by dimobelov** [bludit/bludit#779]
### Describe your problem
Robots plugin Mod
### Expected behavior
example.com/robots.txt
User-agent: *
Allow: /
### Actual behavior
example.com/robots.txt
404 (Not Found)
### Modification
https://github.com/bludit/bludit/blob/b85ecdae89bc7e5527bf6efc792e925987e22430/bl-plugins/robots/plugin.php#L35
```
public function beforeAll()
	{
		$webhook = 'robots.txt';
		if( $this->webhook($webhook) ) {
			
			header('Content-type: text/plain');

			$output  = 'User-Agent: *'.PHP_EOL;
			$output .= 'Allow: /';
			
			echo $output;

			// Terminate the run successfully
			exit(0);
		}
	}
```
### Bludit version
3.0

**Comment by dignajar** [comment]
I updated the plugin, check it https://github.com/bludit/bludit/blob/master/bl-plugins/robots/plugin.php

**Comment by dimobelov** [comment]
Nice!

**Comment by dignajar** [comment]
Implemented in Bludit v3.0.0.
